src/content/tutorials/en/intro-to-p5-strands.mdx

```diff
@@ -0,0 +1,744 @@
+---
+title: "p5.strands: Introduction to Shaders"
+description: An introduction to shaders using p5.strands
+category: p5-strands
+categoryIndex: 0
+featuredImage: ../images/featured/intro-to-p5-strands.jpg
+featuredImageAlt: An abstract, cosmic scene with particles
+relatedContent:
+  references:
+    - en/p5/p5shader
+    - en/p5shader/modify
+    - en/p5shader/inspecthooks
+    - en/p5/basecolorshader/
+    - en/p5/basematerialshader/
+    - en/p5/basestrokeshader/
+    - en/p5/basenormalshader/
+    - en/p5/basefiltershader/
+authors:
+  - Luke Plowden
+---
+
+import EditableSketch from "../../../components/EditableSketch/index.astro";
+import Callout from "../../../components/Callout/index.astro";
+
+## Introduction
+
+In this tutorial, we'll explore **p5.strands**, a new way of writing shaders using JavaScript in p5.js. Shaders are programs that run on the GPU to create visual effects that would be difficult to achieve with traditional rendering.
+
+While many shader effects *could* be created with the p5.js 2D renderer, shaders are more efficient for complex effects and can be consistently applied across multiple objects. Learning shaders is valuable for anyone interested in graphics programming.
+
+We'll create a 3D sketch using 4 different shaders, each illustrating different concepts. The code will be built incrementally throughout the tutorial, with the complete code available at the end.
+
+## What is a Shader?
+
+Shaders are GPU programs written in specialized languages like GLSL for WebGL. There are two main types:
+- **Vertex shaders:** Transform the vertices of shapes (defining where to draw something)
+- **Vertex shaders:** Determine the color of each pixel (defining how to draw something)
+
+<Callout title="Info">
+  A fragment shader is sometimes referred to as a pixel shader.
+</Callout>
+
+A vertex and a fragment shader are both required to render anything in WebGL mode. Fortunately, p5.js provides some default shaders that you can use, so you don't need to write your own shaders. Consider this sketch:

```

Comment:
Minor, phrasing: "Fortunately, p5.js provides some default shaders that you can use, so you don't need to write your own shaders." -> "Fortunately, p5.js provides some default shaders that you can use, so you don't need to write your own."